util/judge.py

```diff
@@ -0,0 +1,188 @@
+import os
+import sys
+from datetime import datetime as dt
+from datetime import timedelta
+from typing import List
+
+import ccxt
+import requests
+from enforce_typing import enforce_types
+from ocean_lib.models.data_nft import DataNFT
+from ocean_lib.ocean import crypto
```

Comment:
Rather than relying on ocean.py here, copy & paste the relevant function from ocean.py's [crypto.py](https://github.com/oceanprotocol/ocean.py/blob/main/ocean_lib/ocean/crypto.py) to here.